test/unit/sqs/index.test.ts

```diff
@@ -91,18 +88,16 @@ describe("SqsQueue tests", () => {
 				done();
 			});
 
-			await queue.stop();
-
-			//delaying to make sure all asynchronous invocations inside the queue will be finished and it will stop
-			queue.start();
-
-			queue.sendMessage(testPayload);
+			queue.stop()
+				//delaying to make sure all asynchronous invocations inside the queue will be finished and it will stop
+				.then(() => queue.start())
+				.then(() => queue.sendMessage(testPayload));
```

Comment:
Didn't we decide to prefer using "await" over "then"?

Reply:
yes, except we can't here because it has a `done` callback and you can't do both without triggering a jest error.  This is the only way to trigger a "throwaway" promise and guarantee the sequential execution of the code.

Reply:
Even if we do:
await queue.stop
await queue.start()
queue.sendMessage()

Reply:
as I said above, we can't have an async function and a done callback at the same time.  Latest jest throws an error because of the race condition. It's one or the other.

Reply:
Ah, we can't make the function itself async